utils/filter.c

```diff
@@ -888,8 +935,15 @@ int setup_trigger_action(char *str, struct uftrace_trigger *tr, char **module,
 		}
 	}
 	if (tr->flags & TRIGGER_FL_CLEAR) {
-		if (orig_flags && (tr->clear_flags == 0))
+		if (orig_flags)
+			/* '@clear' suffix for options other then -T/--trigger */
 			tr->clear_flags = orig_flags;
```

Comment:
This part I thought something like `-F func@clear=depth`.  But it's an advanced usage and maybe better not support it.

Reply:
OK I see, to me clearing a trigger action should only happen in a trigger option.